index.html

```diff
@@ -354,6 +355,103 @@ <h2>Ballerina in Action</h2>
                   </div>
                </div>
             </div>
+             <div id="fourthTab" class="tab-pane fade"  >
+                 <div class="row cdisplayFlex" style="background-color: #f5f5f5;">
+                     <div class="col-lg-7 col-md-12 col-sm-12 focusPane codeBlock noBottomBorder" style="padding:20px;border: 1px #c7c7c7 solid;">
+                         <div class="FullCode" style="background-color: #f5f5f5;">
+                             <div class="highlight">
+                           <pre class="language-ballerina basic ballerina">
+                              <code class="language-ballerina ballerina" style="font-size: 12px !important;">
+import ballerina/grpc;
+
+listener grpc:Listener ep = new (9090);
+
+table&lt;Album&gt; albums = table [
```

Comment:
I can now see ample voice space below the proto file, and I would put each record in one line to reduce that gap. 

If not, can we combine db integration example with this one?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/3728/commits/da7511ce8199a7d3f02b9ad97140aab9aa78b122